docs/3.api/5.kit/3.compatibility.md

```diff
@@ -25,6 +25,7 @@ async function checkNuxtCompatibility(
 interface NuxtCompatibility {
   nuxt?: string;
   bridge?: boolean;
+  builder?: 'vite' | 'webpack' | 'all';
```

Comment:
Setting a single valid builder option makes sense now with only vite + webpack as options, but in future if we have rspack as well, or some other builder, it will be difficult to express the constraint that way.

Maybe instead we could have `vite` or `webpack` as top-level items allowing modules to express not just that they do work (or not!) with them, but additionally which version they require?

What do you think @pi0?